lib/player.rb

```diff
@@ -0,0 +1,61 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'scoring'
+
+module Scrabble
+  class Player < Scoring
+
+    attr_reader :name, :played_words, :player_tiles
+    attr_writer :total_score
```

Comment:
This should **NOT** be a writer.  You're trying to hack the class to make it easier to test, but you've just made it easier to abuse.